src/game/tower_defense/enemy.rs

```diff
@@ -0,0 +1,55 @@
+use crate::game::asset_tracking::LoadResource;
+use bevy::prelude::*;
+use bevy_auto_plugin::auto_plugin::*;
+
+#[auto_register_type]
+#[derive(Resource, Asset, Debug, Clone, Reflect)]
+pub struct EnemyAssets {
+    #[dependency]
+    pub base_skele: Handle<Gltf>,
+}
+
+impl FromWorld for EnemyAssets {
+    fn from_world(world: &mut World) -> Self {
+        let assets = world.resource::<AssetServer>();
+        Self {
+            base_skele: assets.load("models/enemies/Skeleton_Minion.glb"),
```

Comment:
if you checkout the wizard asset resource it's slightly different in terms of loading the glb. It's based off the bevy examples, not sure if its better or not.